### PR TITLE
Login register

### PR DESCRIPTION
**Pull request by HugoEyle**
Create component SignIn

### CODE REVIEW

File: opus/src/components/signin/SignIn.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    password: ''
+  };
+
+  loginUser = () => {
+    axios
+      .post('http://localhost:8000/users/signin', {
+        email: this.state.email,
+        password: this.state.password
+      })
+      .then(function(res) {
```
Comment (ghost) [inline opus/src/components/signin/SignIn.js]: function written in es5 or es6 ? Please use same standard

File: opus/src/components/signin/SignIn.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        console.log(error);
+      });
+  };
+
+  handleChange = event => {
+    const { value, name } = event.target;
+    this.setState({ [name]: value });
+  };
+
+  handleSubmit = event => {
```
Comment (ghost) [inline opus/src/components/signin/SignIn.js]: event parameter never used

File: opus/src/components/signin/SignIn.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    this.setState({
```
Comment (ghost) [inline opus/src/components/signin/SignIn.js:32]: Are you sure you want to clean state in all situations ?

File: opus/src/components/singnup/SignUp.js [hunk 4]
```diff
@@ -10,18 +10,21 @@ export default class SignUp extends Component {
...
-        lastname: this.state.lastnamne,
+        lastname: this.state.lastname,
         society_name: this.state.society_name,
         email: this.state.email,
         password: this.state.password,
-        city: this.state.city
+        city: this.state.city,
+        country: this.state.country
       })
       .then(function(res) {
```
Comment (ghost) [inline opus/src/components/singnup/SignUp.js]: Same review as signin

**Review by ghost — CHANGES_REQUESTED** [review]

File: opus/src/components/signin/SignIn.js (hunk 3 above)
Comment (HugoEyle) [reply opus/src/components/signin/SignIn.js:32]: yes

**Pull request merged**